ebean-core/src/main/java/io/ebeaninternal/server/deploy/BeanDescriptorManager.java

```diff
@@ -445,6 +445,12 @@ public void cacheNotify(TransactionEventTable.TableIUD tableIUD, CacheChangeSet
     return tableToDescMap.get(tableName.toLowerCase());
   }
 
+  @Override
+  public boolean isTableManaged(String tableName) {
+    return tableToDescMap.get(tableName.toLowerCase()) != null
+        || tableToViewDescMap.get(tableName.toLowerCase()) != null;
+  }
```

Comment:
isTableManaged means, a view or an entity with that name exists. This allows you to access the table in a m2m relation. 
See MnyEdge / MnyNode